packages/db/src/Component/Script.tsx

```diff
@@ -626,6 +627,9 @@ function parseDialogueGender(
         female: splitLine(female).map((word) =>
             parseDialogueBasic(region, word, parserDialogueState)
         ),
+        both: splitLine(`${female}{gender}${male}`).map((word) =>
```

Comment:
I wouldn't do another parse like this when both male and female components are available.

Reply:
Understandable, I will try to access the two components, so I won't have to create a new.